### DISCUSSION TITLE
How to understand duplicate metabolites + ion relations

### DISCUSSION THREAD

**Discussion by chi-raag** [shuzhao-li/asari#106] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/317f082c-5100-4b02-9f7d-a7991b825933)

Above is a subset of output from an `asari` run. I'm wondering how to think about Metabolites with slightly varying (MZ, RT) values, but with the same identified ion relation and metabolite name. Is standard practice after an `asari` run to merge these rows, or select the one with the highest peak area as representative, or something else?

Thanks!

**Comment by shuzhao-li** [comment, marked as answer]
Asari by itself is for processing only. 
The default annotation is provided for your convenience, but is considered Level 4 in Schymanski scale.
To get better annotation, one needs authentic compound library, MS2 data etc as descried in https://doi.org/10.1371/journal.pcbi.1011912 .

**Reply by chi-raag** [reply]
Thank you!
